setup.py

```diff
@@ -38,12 +38,19 @@ def get_static_files(path):
     long_description = readme_note + fobj.read()
 
 install_requires = [
-    'tornado>=4.0,<5',
     # https://pagure.io/python-daemon/issue/18
     'python-daemon<2.2.0',
     'python-dateutil>=2.7.5,<3',
 ]
 
+# Tornado >=5 requires updated ssl module so we only allow it for recent enough
+# versions of python (3.4+ and 2.7.9+).
+if sys.version_info[:2] >= (3, 4) or (sys.version_info[:2] == (2, 7)
+                                      and sys.version_info[2]>=9):
+     install_requires.append('tornado>=4.0,<6')
```

Comment:
wouldn't you want this one to be between 5 and 6?

Reply:
I assumed that keeping it at  4.0 as minimum requirement is the correct thing to do as luigi does work with tornado 4.0 and I didn't want to introduce any possible problems that the minimum version is different depending on the python version.

Increasing the minimum version only would make sense if luigi would start using features from tornado 5 which will be complicated due to the split requirement.

Reply:
That's fair. I'm ok with that.

Reply:
@dlstadther Sorry for replying late. I was on a long vacation. Yes I'm OK with this approach.